ReactAndroid/build.gradle

```diff
@@ -6,7 +6,7 @@
 plugins {
     id("com.android.library")
     id("maven")
-    id("de.undercouch.download")
+    id("de.undercouch.download") version "3.4.3"
```

Comment:
same.

Reply:
Version is required because I removed it from top level build.gradle.

Reply:
I mean why not `build.gradle.kts`